tests/format-basic.cmdt

```diff
@@ -0,0 +1,14 @@
+# Test basic formatting
+
+$ echo hello
+1 hello
+
+# Multiple blank lines should be normalized
+
+
+$ echo world 
+1 world
+
+   # Comment with irregular spacing
+$ false
+? 1
```

Comment:
This file doesn't end with a trailing newline. Did you run the formatter you created on these files?

Reply:
still an issue, but you can run the formatter now